src/server_manager/web_app/app.spec.ts

```diff
@@ -193,7 +194,7 @@ class FakeServer implements server.Server {
     this.metricsEnabled = metricsEnabled;
     return Promise.resolve();
   }
-  getServerId() {
+  getMetricsId() {
```

Comment:
This is the shadowboxServerId – so we should come up with a better name. Maybe we revert it back to serverId.

We also need to think through what the names of these are in the views. We should not use `id` so that we don't collide with the HTML id attribute.